### PR TITLE
Inclusion mag phase

### PR DESCRIPTION
**Pull request by dmdmello**
Finished modifications for inclusion of deterministc and non-deterministc magnitude verification.

### CODE REVIEW

File: commit_test [hunk 1]
```diff
@@ -0,0 +1 @@
+commit test
```
Comment (ssvlab) [inline commit_test]: delete file

**Review by ssvlab — COMMENTED** [review]
Review

File: makefile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CC=g++
+CC=g++-4.9
```
Comment (ssvlab) [inline makefile]: -CC=g++

File: bmc/engine/verify_magnitude_and_phase.h [hunk 3]
```diff
@@ -0,0 +1,181 @@
+/**
```
Comment (ssvlab) [inline bmc/engine/verify_magnitude_and_phase.h]: remove this file

File: dsverifier.cpp [hunk 4]
```diff
@@ -0,0 +1,2645 @@
+/*
```
Comment (ssvlab) [inline dsverifier.cpp]: remove this file

File: bmc/core/definitions.h [hunk 5]
```diff
@@ -74,7 +74,9 @@
 #define CONTROLLABILITY	13
 #define OBSERVABILITY	14
 #define LIMIT_CYCLE_STATE_SPACE	15
-#define SAFETY_STATE_SPACE	16
+#define FILTER_MAGNITUDE_DET 16
```
Comment (lennonchaves) [inline bmc/core/definitions.h]: FILTER_MAGNITUDE_DET should be 17
FILTER_MAGNITUDE_NON_DET should be 18

File: benchmarks/filters/filters.h [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+
+#define LOW_PASS 1
+#define HIGH_PASS 2
+
+
+extern implementation impl;
+extern digital_system ds;
+extern filter_parameters prop;
+
+//void choose_filter(int PASS_ID, int IMPLEMENTATION_ID, int DS_ID){
```
Comment (lennonchaves) [inline benchmarks/filters/filters.h]: remove commented code

File: benchmarks/filters/filters.h [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+            .a = { 1.000000000000000,  -1.276247689889290,   2.776103549835551,  -2.645110682028040,   2.795558536616230,  -1.939083083416806,
+      1.253445071015432,  -0.604568772051592,   0.245380454624386,  -0.074737017924551,   0.017001295237337,  -0.002513121328463,
+      0.000185788123619 }, 
+         .a_size = 13
+         };
+      #endif
+
+
+
+     //------------------------------------lp12ESTC-----------------------------------------
```
Comment (lennonchaves) [inline benchmarks/filters/filters.h]: Please, remove this blank code or include the related filters.

File: bmc/core/compatibility.h [hunk 8]
```diff
@@ -37,6 +37,6 @@ void __DSVERIFIER_assert(_Bool expression){
 }
```
Comment (ssvlab) [inline bmc/core/compatibility.h:38]: teste

**Comment by ssvlab** [comment]
teste

**Pull request merged**